queries/zig/highlights.scm

```diff
@@ -93,6 +93,18 @@ field_constant: (IDENTIFIER) @constant
 [
   "allowzero"
   "volatile"
+  "threadlocal"
+  "inline"
+  "noinline"
+  "noalias"
+] @type.qualifier
+
+[
+  "asm"
+  "callconv"
+] @attribute
```

Comment:
that are probably keywords and not `@attributes`. Attributes are custom annotations not added to source code elements not keywords introducing them.

Reply:
zig looks good apart from that

Reply:
llvm never turned into a real tree-sitter grammar (remove?)

Reply:
dart also looks good!

Reply:
> llvm never turned into a real tree-sitter grammar (remove?)

It looks real to me. :thinking:

Reply:
the last time I checked it was just regex on token, without any grammar-like structure that controls their interaction. But nice when it works!